ofl/splinesansmono/METADATA.pb

```diff
@@ -10,7 +10,7 @@ fonts {
   filename: "SplineSansMono[wght].ttf"
   post_script_name: "SplineSansMono-Regular"
   full_name: "Spline Sans Mono Regular"
-  copyright: "Copyright 2022 The Spline Sans Mono Project Authors (https://github.com/SorkinType/SplineSansMono)"
+  copyright: "Copyright 2022 The Spline Sans Project Authors (https://github.com/bghryct/SplineMono)"
```

Comment:
@bghryct Check consistency of copyright mentions in font sources + repo

Reply:
@bghryct The copyright string in the font is still not the same as in the OFL license.